backend/app/views/spree/admin/users/_form.html.erb

```diff
@@ -60,10 +60,13 @@
       <%= f.error_message_on :password %>
     <% end %>
 
-    <%= f.field_container :password do %>
-      <%= f.label :password_confirmation %>
-      <%= f.password_field :password_confirmation, :class => 'fullwidth' %>
-      <%= f.error_message_on :password_confirmation %>
+
+    <% if f.object.respond_to?(:password_confirmation) %>
```

Comment:
Not sure, I'm not really sure which is right in general, I only used `attribute_method?` in the other one cause that's what was already there. 

Hypothetically one could have a user model that provides `password_confirmation` and `password_confirmation=` methods through means other than an AR/AM attribute. In fact, Devise may do that, oops, it may be the other one that's broken. 

I think `respond_to` is right here, not sure what's right there I'll look into it more, I think it might be legit to use different guard methods in a factory for writing tests vs a template in the live app.